pkg/addr/isdas.go

```diff
@@ -48,8 +48,9 @@ type ISD uint16
 
 // ParseISD parses an ISD from a decimal string. Note that ISD 0 is parsed
 // without any errors.
+// @ ensures err != nil ==> err.ErrorMem()
 // @ decreases
-func ParseISD(s string) (ISD, error) {
+func ParseISD(s string) (ret_isd ISD, err error) {
```

Comment:
same comment about the usage of `_`, see my comment below

Reply:
Done.